App.js

```diff
@@ -2,10 +2,20 @@ import React from 'react';
 import {StyleSheet} from 'react-native';
 
 import Tabs from "./src/navigation/Tabs";
-export default function App() {
+import Profile from "./src/components/Profile/Profile";
+import {ApolloClient, ApolloProvider, InMemoryCache} from "@apollo/client";
+
 
+export default function App() {
+  const client = new ApolloClient({
```

Comment:
Please create separate config for such things as URL and so on. :) You need one source of truth. 

Also in a future you will need to make a lot of changes in InMemoryCache(), so I would suggest you also move ApolloClient initialization into separate file. :)